spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
```

Comment:
This was suggested also by @mbj, and I tend to use `described_class` myself, but that is not the convention of the specs to date. @backus?